assets/json/assets.json

```diff
@@ -472,13 +472,6 @@
                                 "scale": 0.9
                             },
                             "children": {
-                                "shipsegments": {
```

Comment:
why is this unnecessary?

Reply:
It was just a dummy node for the scene graph code to add nodes to. I turned that dummy node into its own class (`ShipSegmentWrap`), so we don't need this in the JSON anymore.